gdcdictionary/schemas/treatment.yaml

```diff
@@ -130,8 +199,14 @@ properties:
     $ref:
       - "_terms.yaml#/route_of_administration/common"
     enum:
+      - Intrahepatic
+      - Intramuscular
+      - Intraperitoneal
       - Intrathecal
+      - Intratumor
       - Intravenous
+      - Intravesical
+      - Subcutaneous
       - Oral
       - Not Reported
 
```

Comment:
just noting we didn't do array of enums for this prop.

Reply:
Fixed!

Reply:
looks good